src/Goerli-DssSpell.sol

```diff
@@ -32,10 +36,16 @@ interface TeleportOracleAuthLike {
     function removeSigners(address[] calldata) external;
 }
 
-contract DssSpellAction is DssAction, DssSpellCollateralAction {
+interface RwaUrnLike {
+    function lock(uint256) external;
+    function draw(uint256) external;
+}
 
+contract DssSpellAction is DssAction, DssSpellCollateralAction {
     // Provides a descriptive tag for bot consumption
-    string public constant override description = "Goerli Spell";
+    // This should be modified weekly to provide a summary of the actions
+    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/969f04cfec25e56791fbe4503bcbe2df7a58df1e/governance/votes/Executive%20vote%20-%20July%2029%2C%202022.md -q -O - 2>/dev/null)"
```

Comment:
maybe this should be removed or the first comment changed to specify default behaviour on goerli.

Reply:
Done